src/wireviz/wv_helper.py

```diff
@@ -75,8 +75,8 @@ def html_caption(image):
 
 def html_size_attr(image):
     # Return Graphviz HTML attributes to specify minimum or fixed size of a TABLE or TD object
-    return ((f' width="{image.width}"'   if image.width else '')
-        +   (f' height="{image.height}"' if image.height else '')
+    return ((f' width="{image.width}"'   if image.width is not None else '')
+        +   (f' height="{image.height}"' if image.height is not None else '')
```

Comment:
See comment above, this change is probably not needed?

Reply:
You probably mean: _replaced by `if image.X`?_ (like it was earlier)